build-tools/package.json

```diff
@@ -100,6 +101,36 @@
 	"dependenciesComments": {
 		"@fluidframework/build-tools": "Provides fluid-build which is used by the `build:fast` script. To allow this to work when the workspace version of build-tools has not been built yet, a prepackaged version is depended on. If this self-dependency becomes problematic, the `build` script can be used which does not require it."
 	},
+	"fluidBuild": {
+		"tasks": {
+			"build": {
+				"dependsOn": [
+					"^build",
+					"generate:packageList",
+					"syncpack:deps",
+					"syncpack:versions"
+				],
+				"script": false
+			},
+			"checks": {
+				"dependsOn": [
+					"check:format",
+					"generate:packageList",
+					"syncpack:deps",
+					"syncpack:versions"
+				],
+				"script": false
+			},
+			"clean": {
+				"dependsOn": [
+					"^clean",
```

Comment:
I questioned for a second if this pattern for task dependencies would work between the root package and the rest of the packages in the release group. Based on what I see for client it must be working, I'm just a bit surprised, I thought it would only care about the explicit package dependencies from this `package.json` file, but then I assume we have logic to make it work throughout the release group somehow?

Reply:
Yeah, the root tasks are a bit unique. There's a "virtual" dependency between the root and all packages in the workspace, so the `^clean` string in this case means "all clean tasks for all dependencies = packages in this release group."